Bottom: c649c7fb44e510d4d0b26a902a2cea337070b445
Top:    0cba386a52bcae3ea43d0d5b308a1ce4ec1d6af2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:34:07 +0100

Refresh of refer-to-git-sources-not-cvs-w

---

diff --git a/src/file.c b/src/file.c
index 639c971..2704ede 100644
--- a/src/file.c
+++ b/src/file.c
@@ -544,7 +544,7 @@ WritePCBDataHeader (FILE * FP)
    * ************************** README *******************
    */
 
-  fprintf (FP, "\n# To read pcb files, the pcb version (or the cvs source date) must be >= the file version\n");
+  fprintf (FP, "\n# To read pcb files, the pcb version (or the git source date) must be >= the file version\n");
   fprintf (FP, "FileVersion[%i]\n", PCB_FILE_VERSION);
 
   fputs ("\nPCB[", FP);
